Makefile

```diff
@@ -53,7 +53,7 @@ clean: ## clean the repository
 
 dev_install: ## set up the repo for active development
 	${PIP} install -e .[dev]
-	${PYTHON} -m jupyter serverextension enable --py jupyterfs
+	${PYTHON} -m jupyter serverextension enable --py jupyterfs.extension
```

Comment:
This isn't necessary as the activation is exposed in the package init

Reply:
The way we had it was "mixed", in that you could enable *either* `jupyterfs` or `jupyterfs.extension` and in both cases the serverextension would work, but not perfectly (eg in current master, if you enable `jupyterfs.extension` the version won't get picked up correctly).

 I want to clean this up by picking one or the other. I figure putting the serverextension-specific stuff in `jupyterfs.extension` is slightly nicer/more explicit. Also supports the off-chance someone has a use case for something in our backend package other than the serverextension